src/core/apim_checkout.tf

```diff
@@ -346,3 +346,12 @@ resource "azurerm_api_management_api_operation_policy" "transaction_authorizatio
     ecommerce_vpos_psps_list = var.ecommerce_vpos_psps_list
   })
 }
+
+resource "azurerm_api_management_api_operation_policy" "transaction_activation_request" {
```

Comment:
to delete, because they refer to the old deprecated APIs

Reply:
All this file is to delete or only this resource?